settings.py

```diff
@@ -94,6 +94,8 @@
 
 TIMER_PROXY_PASS = "http://{}:{}".format(PROXY_HOST, TIMER_PORT)
 
+INCLUDES = ""
```

Comment:
This is too vague. The value must and could be valid nginx config. I would rename it so (ngnix_conf) or made a list of include _files_.

Reply:
I think we can follow up on this in a later patch. The idea here is that INCLUDES is simply text that would be added to the nginx config when necessary.